src/main/java/edu/kis/powp/jobs2d/features/driverTransofrmation/modifiers/VerticalFlipModifier.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.powp.jobs2d.features.driverTransofrmation.modifiers;
+
+import edu.kis.powp.jobs2d.features.driverTransofrmation.TransformationModifier;
+
+public class VerticalFlipModifier implements TransformationModifier {
+    public VerticalFlipModifier() {
+    }
+
+    @Override
+    public int[] modify(int x, int y) {
+        return new int[]{ x, (y * -1)};
+    }
```

Comment:
this looks like a special case of ScalingModifier. Instead of VerticalFlipModifier (and other similar), please add a factory of ScalingModifier preset instances.